README.md

```diff
@@ -4,6 +4,11 @@
 
 ![CodeQC](https://github.com/Novartis/torchsurv/actions/workflows/codeqc.yml/badge.svg?branch=main)
 ![Docs](https://github.com/Novartis/torchsurv/actions/workflows/docs.yml/badge.svg?branch=main)
+[![arXiv](https://img.shields.io/badge/arXiv-2404.10761-f9f107.svg)](https://arxiv.org/abs/2404.10761)
+[![Documentation](https://img.shields.io/badge/GitBook-Docu-lightblue)](https://opensource.nibr.com/torchsurv/)
+[![PyPI version](https://badge.fury.io/py/torchsurv.svg)](https://badge.fury.io/py/torchsurv)
```

Comment:
Can be done also with 
https://img.shields.io/pypi/v/torchsurv?label=pypi%20package